docs/about/index.rst

```diff
@@ -8,6 +8,31 @@ About
 Mission
 -------
 
+Fairlearn is an open source project that seeks to empower data scientists and
+developers to assess and improve fairness of AI systems.
+The project aspires to include:
+
+- A Python library of fairness metrics and mitigation algorithms
+- Tools for (interactive) fairness assessment (e.g., plotting, dashboards, etc.)
+- Educational materials (comprehensive user guide, detailed case studies,
+  Jupyter notebooks, white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.
+Because there are many complex sources of unfairness&mdash;some societal and
+some technical&mdash;it is not possible to fully "debias" a system or to
+guarantee fairness.
+Our goal is to enable humans to assess fairness-related harms, review the
+impacts of different mitigation strategies and then make trade-offs
+appropriate to their scenario.
+
+Fairlearn is currently driven by Microsoft Research and Azure with a substantial
+involvement of external contributors and an intention to eventually transition to a
+community-driven open source project under a neutral organization.
```

Comment:
@MiroDudik says over email that this likely won't happen in 2020 (the next NumFocus deadline is 10/15/20), and the project will explore open governance models and transitioning to a neutral organization sometime in 2021.

Reply:
Closing the comment since this actually happened :-)